pyturbo/systems/nozzle/nozzle_aero.py

```diff
@@ -66,14 +68,23 @@ def setup(self, FluidLaw=IdealDryAir):
         self.add_inward("pamb", 101325.0, unit="Pa", desc="ambient static pressure")
 
         # geom
-        self.add_inward("area_in", 10.0, unit="m**2", desc="inlet aero section")
-        self.add_inward("area_exit", 10.0, unit="m**2", desc="exit aero section")
-        self.add_inward("area", 1.0, unit="m**2", desc="choked/exit area")
+        self.add_inward("area_in", 0.0625 * np.pi, unit="m**2", desc="inlet aero section")
+        self.add_inward("area_exit", 0.0225 * np.pi, unit="m**2", desc="exit aero section")
+        self.add_inward("area", 0.0225 * np.pi, unit="m**2", desc="choked/exit area")
+        self.add_inward("gamma", 1.4, unit="", desc="Heat capacity ratio")
+        self.add_inward("rho_1", 1.2, unit="kg/m**3", desc="Fluid density at inlet")
 
         # outwards
-        self.add_outward("ps", 0.0, unit="pa", desc="static pressure at throat")
-        self.add_outward("mach", 0.0, unit="", desc="mach at throat")
-        self.add_outward("speed", 0.0, unit="m/s", desc="fluid flow speed at throat")
+        self.add_outward("Ps1", 0.0, unit="pa", desc="static pressure at inlet")
+        self.add_outward("Ps2", 0.0, unit="pa", desc="static pressure at outlet")
+        self.add_outward("Ps_crit", 0.0, unit="pa", desc="critical static pressure at throat")
+        self.add_outward("Ts1", 0.0, unit="pa", desc="static pressure at inlet")
+        self.add_outward("Ts2", 0.0, unit="pa", desc="static pressure at outlet")
+        self.add_outward("M1", 0.0, unit="", desc="mach at inlet")
+        self.add_outward("M2", 0.0, unit="", desc="mach at outlet")
+        self.add_outward("mach", 0.0, unit="", desc="mach at outlet")
+        self.add_outward("rho_2", 1.2, unit="kg/m**3", desc="Fluid density at outlet")
+        self.add_outward("speed", 0.0, unit="m/s", desc="fluid flow speed at outlet")
```

Comment:
do not add outwards unless they are necessary (mainly, used for postprocessing or for connections).
please also respect the conventional case

Reply:
unnecessary outwards have been removed and conventional case has been applied